src/lib/components/ratio/ColorIssues.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+
+  import ExternalLink from '$lib/components/util/ExternalLink.svelte';
+
+  export let pass: boolean;
```

Comment:
@jamesnw Does this change (in https://github.com/oddbird/oddcontrast/pull/146/commits/af6b86db0f0c734ae876a45aabfbaf12b59c3e43) make sense? I figure this should probably use the same `pass` logic (and might as well use the same reactive boolean) as the ratio display.

Reply:
I'm not opposed to reuse, but I wanted to use the AA Normal contrast ratio, where `pass` uses AA Large- I think Normal makes sense, given the text size.